src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -147,4 +185,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(newCourse);
     response.sendRedirect("/index.html");
   }
+
+  // TODO: Adapt this to quarter-system schools and potentially include summer quarter.
+  private String getPrevTerm(String term) {
+    String season = term.split(" ")[0];
+    int year = Integer.parseInt(term.split(" ")[1]);
+    if (season.equals("Fall")) {
+      return "Spring " + String.valueOf(year);
+    } else { // Season is "Spring".
+      return "Fall " + String.valueOf(year - 1);
+    }
+  }
+
+  private String getNextTerm(String term) {
+    String season = term.split(" ")[0];
+    int year = Integer.parseInt(term.split(" ")[1]);
+    if (season.equals("Fall")) {
+      return "Spring " + String.valueOf(year + 1);
+    } else { // Season is "Spring"
+      return "Fall " + String.valueOf(year);
+    }
```

Comment:
You and Dagm should get together on this code and separate it out into a utility.
Whatever you end up doing, you'll want to be consistent.

Reply:
In Dagm's JS he's currently roughly doing the same thing as me (I think he's just getting the date of a term to use as a timestamp), but will sync up with him on this to make sure we don't have any overlapping code